bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,222 @@
+# MagentaTV Binding (aka TelekomTV aka EntertainTV)
+
+---
+
+## Release: 2.5-pre5
+
+This Binding allows controlling the Deutsche Telekom Magenta TV receiver series Media Receiver MR4xx and MR2xx (Telekom NGTV / Huawei Envision platform). 
+The Binding does NOT support MR3xx/1xx (old Entertain system based on Microsoft technology)!
+
+The binding has been tested with the EntertainTV service as well as the new MagentaTV service (launched in 10/2018).
+
+This include device discovery, sending keys from the remote and also receiving program events.
+
+This binding replaces the former versions EntertainTV and TelekomTV bindings.
+
+---
+
+Please check out the [openHAB community thread](https://community.openhab.org/t/magentatv-entertaintv-binding-for-deutsche-telekom-mr-3xx-and-4xx) and discuss your ideas, requests and technical problems with the community.
+
+---
+
+### Supported Models
+
+* Deutsche Telekom Media Receiver MR401B - fully supported
+* Deutsche Telekom Media Receiver MR201  - fully supported
+* Deutsche Telekom Media Receiver MR400  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR200  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR3xx  - NOT supported (different platform)
+* Deutsche Telekom Media Receiver MR1xx  - NOT supported (different platform)
+
+
+### Receiver Standby Mode
+
+The Media receiver has 3 different standby modes, which could be selected in the receiver's settings.
+
+- Standby - full standby, receiver active all the time
+- Suspend/Resume - The receiver goes to sleep mode, but could be awaked by a Wake-on-LAN packet
+- Shutdown - Powering off will shutdown the receiver, can be awaked only with the power button
+
+Standby provides the best results, because the binding could wakeup the receiver (Power On/Off). Suspend/Resume would require a Wake-on-LAN, which could be done, but is currently not implemented. Shutdown turns the receiver off, which requires a manual PowerOn.
+
+There is no way to detect the "display status" of the receiver. The binding detects PowerOff with the MR401B/MR201 by listening to UPnP events, but can't verify the status when started. You need to take care on the current status if you power on/off the receiver from scenes. Check the current status before sending the POWER button, because POWER is a toggle, not ON or OFF (see sample rules).
+
+### Supported Things
+
+| Thing    | Type                                                           |
+| ---------| -------------------------------------------------------------- |
+| receiver | A MagentaTV Receiver, the binding supports multiple receivers. |
+
+
+## Discovery
+
+The auto discovery starts when the binding is loaded. UPnP will be used to discover receivers on the local network. Based on UPnP all nessesary parameters are auto-discovered. The receiver needs to be powered on to get discovered.
+
+# Thing Configuration
+
+## Using PaperUI
```

Comment:
Try to not differentiate between the configuration ways. Just list the parameters that are required for a thing and possibly describe how to get hold of their values.

Reply:
please check latest update (once checked in)